index.html

```diff
@@ -1,556 +1,290 @@
-				</a>
-				<br>
-				<h4 class="mobile-learn-padding" data-i18n="dgc-windows-wallet-modal-getting-started">Learn how to set up your wallet with the <a href="/getting-started" target="_blank">Getting Started Guide</a>.</h4>
-			</div>
-		</div>
-	</div>
-</div>
-
-<!-- Google Analytics -->
-<script>
-(function(i,s,o,g,r,a,m){
-	  i['GoogleAnalyticsObject']=r;
-	  i[r]=i[r]||function(){
-		(i[r].q=i[r].q||[]).push(arguments)}
-		,i[r].l=1*new Date();
-	  a=s.createElement(o),
-		m=s.getElementsByTagName(o)[0];
-	  a.async=1;
-	  a.src=g;
-	  m.parentNode.insertBefore(a,m)
-	}
-	)(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-	ga('create', 'UA-28461406-3', 'dogecoin.com');
-	ga('send', 'pageview');
-</script>
-
-<!-- AddThis Smart Layers -->
-<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-529f00d46dc73a67" type="text/javascript"></script>
-<script type="text/javascript">
-addthis.layers({
-      'theme' : 'transparent',
-      'share' : {
-        'position' : 'left',
-        'numPreferredServices' : 5
-      }
-
-    }
-                  );
-</script>
-<script src="/js/dogecoin.min.js"></script>
-
+<body id="page-top">
+    <!-- Navigation-->
+    <nav class="navbar navbar-expand-lg bg-secondary text-uppercase fixed-top" id="mainNav">
+        <div class="container-fluid">
+            <a class="navbar-brand js-scroll-trigger" href="#page-top"><img src="assets/img/dogecoin-300.png">&nbsp;Dogecoin</a>
+            <button class="navbar-toggler navbar-toggler-right text-uppercase font-weight-bold bg-primary text-secondary rounded" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                <span data-i18n="dgc-menu">Menu</span>
+                <i class="fas fa-bars"></i>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarResponsive">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#wallets">Wallets</a></li>
+                    <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#about"><span data-i18n="dgc-what-is-doge">What is dogecoin</span></a></li>
+                    <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#start"><span data-i18n="dgc-get-started-now">How to get started</span></a></li>
+                    <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="guide.html"><span data-i18n="dgc-guide">Guide</span></a></li>
+                    <li class="nav-item mx-0 mx-lg-1"><a href="" class="nav-link py-3 px-0 px-lg-3 rounded text-white curPrice_USD"></a></li>
```

Comment:
this needs to be removed

Reply:
i think you are referring to the curPrice_USD element. removed that now. all other elements are part of the menu and shouldn't be removed